pytorch_amazon_network_analysis/README.md

```diff
@@ -0,0 +1,89 @@
+#PyTorch x Amazon product reviews = :heart:
+
+Purpose of this  **Jupyter Notebook** is to make it easy to start **exploring and learning**
+about **Temporal Graph Networks** and **GNNs** as part of **Memgraph** and in general.
+
+
+## Table of contents
+ * [Introduction to repo](#introduction)
+ * [What are Graph Neural Networks and TGN](#what-are-gnns)
+ * [Dataset visualized](#dataset-visualized)
+ * [About Amazon product dataset](#about-dataset)
+ * [Setup](#setup)
+ * [Learning materials](#learning-materials)
+
+##Introduction
+This directory is an example of how to use **[Temporal Graph Networks](https://memgraph.com/docs/mage/query-modules/python/tgn)** for the
+link prediction on **[Amazon product graph](http://snap.stanford.edu/data/amazon/productGraph/)** dataset.
+
+In this Jupyter Notebook  we run **[Memgraph](https://memgraph.com/docs/memgraph/)** inside 
+**[Docker](https://www.docker.com/)** and use **[GQLAlchemy](https://memgraph.com/docs/gqlalchemy/)** as a link between 
+Graph Database objects and Python objects. To start you will need **Docker** and you will need to download 
+**Docker image** of **[Memgraph Platform](https://memgraph.com/download)**
+
+
+## What are GNNs
+**Graph neural networks** present a neural network models dealing with convolutions over graphs. It all started
+with **[Graph convolutional networks](https://arxiv.org/abs/1609.02907)** and idea of researchers to start exploring signals on graphs.
+Actually those ideas started way before with [spectral clustering](https://arxiv.org/pdf/0711.0189.pdf), 
+but after several years of research and polishing signal exploration over graph, **graph neural networks** were born.
```

Comment:
```suggestion
Those ideas first yielded [spectral clustering](https://arxiv.org/pdf/0711.0189.pdf), 
but after several years of researching signals in graphs **graph neural networks** were born.
```
Suggesting this so that we don't have two consecutive sentences that start with "X started".